NEWS

```diff
@@ -7,13 +7,177 @@
 ##
 #
 
-## Current
+## 1.12.0 RC1 (December 2, 2021)
 ### Features:
+#### Core
+* Added initial support for Go language bindings
+* Added memory invalidation on error detection
+* Added threshold for endpoint connection matching in UCP
+* Added new objects to VFS (md, component, log_level, etc)
+* Added configuration variable to specify what loadable modules are needed
+* Added sigaction-override configuration to disable sigaction overriding
 #### UCP
-* Added API for querying UCP library attributes
+* Added address versioning to correctly preserve wire compatibility since v1.11.0
+* Added new sockaddr private data format
+* Enabled rendezvous and tag sync for all cases with error handling
+* Added iov zcopy support to RMA operations
+* Added usage of mpool set for unexpected eager message to reduce memory consumption
```

Comment:
Reduced memory usage of unexpected messages by fitting receive buffer size to packet size

Reply:
missed?